ds3-sdk/src/main/java/com/spectralogic/ds3client/networking/FailedRequestUsingMgmtPortException.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * ******************************************************************************
+ *   Copyright 2014-2015 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+package com.spectralogic.ds3client.networking;
+
+import com.google.common.base.Joiner;
+import com.google.common.collect.ImmutableList;
+import com.spectralogic.ds3client.models.Error;
+
+import java.io.IOException;
+import java.util.List;
+
+public class  FailedRequestUsingMgmtPortException extends  FailedRequestException {
+
+    public final static int MGMT_PORT_STATUS_CODE = 404;
```

Comment:
The static members? I compare them in other classes (AbstractResponse and Ds3ClientTest)